pkg/policy/rule_test.go

```diff
@@ -234,6 +234,160 @@ func (ds *PolicyTestSuite) TestL4Policy(c *C) {
 	c.Assert(state.selectedRules, Equals, 0)
 }
 
+func (ds *PolicyTestSuite) TestMergeL7Policy(c *C) {
+	toBar := &SearchContext{To: labels.ParseSelectLabelArray("bar")}
+	toFoo := &SearchContext{To: labels.ParseSelectLabelArray("foo")}
+
+	fooSelector := []api.EndpointSelector{
+		api.NewESFromLabels(labels.ParseSelectLabel("foo")),
+	}
+	rule1 := &rule{
+		Rule: api.Rule{
+			EndpointSelector: api.NewESFromLabels(labels.ParseSelectLabel("bar")),
+			Ingress: []api.IngressRule{
+				{
+					ToPorts: []api.PortRule{{
+						Ports: []api.PortProtocol{
+							{Port: "80", Protocol: "tcp"},
```

Comment:
Let's use the new constants instead of "tcp": `api.ProtoTCP`

Reply:
Sure, I'll go through and fix these up.